schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -187,46 +190,224 @@
             "2015-06-15"
           ]
         },
+        "location": {
+          "type": "string"
+        },
         "properties": {
-          "properties": {
-            "publisher": {
-              "type": "string",
-              "minLength": 1,
-              "description": "Microsoft.Compute/extensions - Publisher"
-            },
-            "type": {
-              "type": "string",
-              "minLength": 1,
-              "description": "Microsoft.Compute/extensions - Type"
+          "oneOf": [
```

Comment:
If we keep the old definition, we'll end up exporting partial extensions since the settings property isn't fully defined. Since we're greenlighting three extensions and not others, it doesn't make sense to fully support three and partially support all others.